homeassistant/components/soundtouch/media_player.py

```diff
@@ -50,137 +49,58 @@
     "STOP_STATE": STATE_OFF,
 }
 
-DATA_SOUNDTOUCH = "soundtouch"
 ATTR_SOUNDTOUCH_GROUP = "soundtouch_group"
 ATTR_SOUNDTOUCH_ZONE = "soundtouch_zone"
 
-SOUNDTOUCH_PLAY_EVERYWHERE = vol.Schema({vol.Required("master"): cv.entity_id})
-
-SOUNDTOUCH_CREATE_ZONE_SCHEMA = vol.Schema(
-    {vol.Required("master"): cv.entity_id, vol.Required("slaves"): cv.entity_ids}
-)
-
-SOUNDTOUCH_ADD_ZONE_SCHEMA = vol.Schema(
-    {vol.Required("master"): cv.entity_id, vol.Required("slaves"): cv.entity_ids}
-)
-
-SOUNDTOUCH_REMOVE_ZONE_SCHEMA = vol.Schema(
-    {vol.Required("master"): cv.entity_id, vol.Required("slaves"): cv.entity_ids}
-)
-
-DEFAULT_NAME = "Bose Soundtouch"
-DEFAULT_PORT = 8090
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    }
+PLATFORM_SCHEMA = vol.All(
+    cv.deprecated(CONF_PORT),
```

Comment:
Everything is deprecated, not just port.

Reply:
I went with what I found in other components: logging the migration notice any deprecating the whole config section in the next version? Should I completely deprecate it now?

Reply:
you already print a deprecation warning inside `async_setup_platform` so this single key deprecation is not necessary.